### PR TITLE
Add some additional libraries

### PR DESCRIPTION
**Pull request by cldwalker**
Hi @borkdude. I'd been meaning to add some libraries I noticed that were bb compatible related to eql. This PR also
also adds the libraries for #1137 and #1128 and adds integrant tests that were accidentally disabled in https://github.com/babashka/babashka/pull/1109/commits/685e85a1d14f7bd71e32e619bda0d40519aeae9d. For the new libraries, there were a couple minor test patches in eql and pyramid around differing meta behavior for bb in case you're interested.

As an aside, seeing that integrant is bb-compatible, I briefly looked to see if the new kit-clj, which is based on integrant, would be bb compatible. While kit-core could be made compatible with a fork, the [kit-generator lib](https://github.com/kit-clj/kit/blob/master/libs/kit-generator/deps.edn) has too many non-compatible deps at this time

Cheers, Gabriel

---
Please answer the following questions and leave the below in as part of your PR.

- [x] I have read the [developer documentation](https://github.com/babashka/babashka/blob/master/doc/dev.md).

- [x] This PR corresponds to an [issue with a clear problem statement](https://github.com/babashka/babashka/blob/master/doc/dev.md#start-with-an-issue-before-writing-code).

- [x] This PR contains a [test](https://github.com/babashka/babashka/blob/master/doc/dev.md#tests) to prevent against future regressions

- [-] I have updated the [CHANGELOG.md](https://github.com/babashka/babashka/blob/master/CHANGELOG.md) file with a description of the addressed issue.

### CODE REVIEW

File: doc/libraries.csv [hunk 1]
```diff
@@ -12,17 +12,20 @@ clj-commons/clj-yaml,https://github.com/clj-commons/clj-yaml
 clj-commons/multigrep,https://github.com/clj-commons/multigrep
 clojure-csv/clojure-csv,https://github.com/davidsantiago/clojure-csv
 clojure-term-colors/clojure-term-colors,https://github.com/trhura/clojure-term-colors
+com.exoscale/lingo,https://github.com/exoscale/lingo
 com.github.seancorfield/honeysql,https://github.com/seancorfield/honeysql
 com.grammarly/omniconf,https://github.com/grammarly/omniconf
 com.stuartsierra/component,https://github.com/stuartsierra/component
 com.stuartsierra/dependency,https://github.com/stuartsierra/dependency
+com.wsscode/cljc-misc,https://github.com/wilkerlucio/cljc-misc
```
Comment (cldwalker) [inline doc/libraries.csv:20]: @wilkerlucio Hi. I added a couple of your libraries that are bb compatible. It's easy to add new libraries nowadays. I did this one with `script/add-libtest.clj com.wsscode/pathom -t`

**Comment by borkdude** [comment]
I notice this in pyramid:

```
77:   ([m [k ek] f x]
78:    (let [em (get m k {})
79:          v (get em ek)]
80:      #?(:clj (fast-assoc m k
81:                          (fast-assoc em ek
                             ^--- No matching method assoc found taking 2 args for class clojure.lang.PersistentArrayMap
82:                                      (f v x)))
83:         :cljs (assoc m k
84:                      (assoc em ek
85:                             (f v x)))))))
```

cc @bsless @lilactown: if you care about bb compatibility for this lib, then I recommend introducing a `:bb` reader conditional _before_ the `:clj` reader conditional which does the same as the `:cljs` branch here.

**Comment by cldwalker** [comment]
Ah. I should probably start running non-jvm tests locally. I just tried that last commit before going to bed. I'm happy to pull out pyramid and revert the last commit now or wait a bit to hear if they're interested in adding bb compatibility

**Comment by cldwalker** [comment]
@lilactown Happy to add a pull request like the above if you're open to bb compatibility for pyramid

**Comment by lilactown** [comment]
I'd like to support bb in pyramid, but i'm not sure how best to do that. I think first step would be getting the tests in https://github.com/lilactown/pyramid running under babashka. @cldwalker I would certainly accept a PR that added a bb test runner that runs in github. Any necessary changes to help them pass would also be appreciated.

**Comment by borkdude** [comment]
Feel free to split the PR with the stuff that's already good.

File: src/babashka/impl/classes.clj [hunk 2]
```diff
@@ -86,6 +86,8 @@
     {:methods [{:name "getHostAddress"}]}
     clojure.lang.IFn
     {:methods [{:name "applyTo"}]}
+    clojure.lang.MapEntry
+    {:methods [{:name "create"}]}
```
Comment (borkdude) [inline src/babashka/impl/classes.clj:90]: Why do we need this one again?

Comment (cldwalker) [reply src/babashka/impl/classes.clj]: For pyramid. Reverted it and can pull it in again when needed

**Pull request merged**